app/policies/v2/system_policy.rb

```diff
@@ -23,6 +23,10 @@ def destroy?
       match_account? && match_group?
     end
 
+    def os_versions?
+      true
```

Comment:
```suggestion
      match_account? && match_group?
```

Reply:
No, as this is applied on a collection and there you don't have individual systems to match against, see `#index`